catalogue_of_life/Loader.java

```diff
@@ -140,24 +141,48 @@ private static void giveNamesUniqueID(Path dataDirectory) throws IOException {
 
     private static void separateTaxonDistributions(Path dataDirectory) throws IOException {
         TsvParser distributionsParser = createTsvParser(dataDirectory.resolve("Distribution.tsv"));
-        TsvWriter marineRegionsWriter = createTsvWriter(dataDirectory.resolve("MarineDistribution.tsv"));
-        TsvWriter describedRegionsWriter = createTsvWriter(dataDirectory.resolve("DescribedDistribution.tsv"));
+        TsvWriter marineRegionsWriter = createTsvWriter(dataDirectory.resolve("MarineRegions.tsv"));
+        TsvWriter describedRegionsWriter = createTsvWriter(dataDirectory.resolve("DescribedRegions.tsv"));
+        TsvWriter marineDistributionsWriter = createTsvWriter(dataDirectory.resolve("MarineDistribution.tsv"));
+        TsvWriter describedDistributionsWriter = createTsvWriter(dataDirectory.resolve("DescribedDistribution.tsv"));
```

Comment:
`*Regions` is for `*-region` entities, `*Distribution` is for `*-distribution` relations.